src/pages/BookDetailsPage/BookDetails/ReviewSection/ReviewCollection/ReviewCard/index.tsx

```diff
@@ -13,21 +15,33 @@ type Props = {
 
 const ReviewCard = ({
   reviewer,
+  reviewerId,
   rating,
   recommendation,
   comment,
   ratingsCount,
 }: Props) => {
+  const { userProfile } = useAuthContext();
   const recommendationString = recommendation?.join(", ");
+
+  const isYourReview = () => reviewerId === userProfile?.email;
```

Comment:
This can be a simple boolean const, no need for a method
```suggestion
  const isYourReview = reviewerId === userProfile?.email;
```